test/runtests.jl

```diff
@@ -32,6 +32,10 @@ end
         include(joinpath("mlj", "regression.jl"))
     end
 
+    @testset "MLJ update interface" begin
+        include(joinpath("mlj", "update.jl"))
```

Comment:
You have a `true` at the end of the module but you're not calling `@test` on the `include`?

Reply:
Yes I think I discovered it wasn't necessary and forgot to dump the true's